symbols/arm7.yml

```diff
@@ -1,88 +1,153 @@
+        NA: 0x37FDCEC
+        JP: 0x37FDCEC
       description: |-
-        Enables processor interrupts by clearing the i flag in the program status register (cpsr).
+        Copy of the ARM9 function. See arm9.yml for more information.
         
         return: Old value of cpsr & 0x80 (0x80 if interrupts were disabled, 0x0 if they were already enabled)
     - name: EnableIrqFlag
       address:
-        EU: 0x2385EE8
-        NA: 0x2385EE8
-        JP: 0x2385EE8
+        EU: 0x37FDD00
+        NA: 0x37FDD00
+        JP: 0x37FDD00
       description: |-
-        Disables processor interrupts by setting the i flag in the program status register (cpsr).
+        Copy of the ARM9 function. See arm9.yml for more information.
         
         return: Old value of cpsr & 0x80 (0x80 if interrupts were already disabled, 0x0 if they were enabled)
     - name: SetIrqFlag
       address:
-        EU: 0x2385EFC
-        NA: 0x2385EFC
-        JP: 0x2385EFC
+        EU: 0x37FDD14
+        NA: 0x37FDD14
+        JP: 0x37FDD14
       description: |-
-        Sets the value of the processor's interrupt flag according to the specified parameter.
+        Copy of the ARM9 function. See arm9.yml for more information.
         
         r0: Value to set the flag to (0x80 to set it, which disables interrupts; 0x0 to unset it, which enables interrupts)
         return: Old value of cpsr & 0x80 (0x80 if interrupts were disabled, 0x0 if they were enabled)
     - name: EnableIrqFiqFlags
       address:
-        EU: 0x2385F14
-        NA: 0x2385F14
-        JP: 0x2385F14
+        EU: 0x37FDD2C
+        NA: 0x37FDD2C
+        JP: 0x37FDD2C
       description: |-
-        Disables processor all interrupts (both standard and fast) by setting the i and f flags in the program status register (cpsr).
+        Copy of the ARM9 function. See arm9.yml for more information.
         
         return: Old value of cpsr & 0xC0 (contains the previous values of the i and f flags)
     - name: SetIrqFiqFlags
       address:
-        EU: 0x2385F28
-        NA: 0x2385F28
-        JP: 0x2385F28
+        EU: 0x37FDD40
+        NA: 0x37FDD40
+        JP: 0x37FDD40
       description: |-
-        Sets the value of the processor's interrupt flags (i and f) according to the specified parameter.
+        Copy of the ARM9 function. See arm9.yml for more information.
         
         r0: Value to set the flags to (0xC0 to set both flags, 0x80 to set the i flag and clear the f flag, 0x40 to set the f flag and clear the i flag and 0x0 to clear both flags)
         return: Old value of cpsr & 0xC0 (contains the previous values of the i and f flags)
     - name: GetProcessorMode
       address:
-        EU: 0x2385F40
-        NA: 0x2385F40
-        JP: 0x2385F40
+        EU: 0x37FDD58
+        NA: 0x37FDD58
+        JP: 0x37FDD58
       description: |-
         Gets the processor's current operating mode.
         
-        See https://problemkaputt.de/gbatek.htm#armcpuflagsconditionfieldcond
```

Comment:
Is this not applicable? Not sure why we're deleting it. It seems useful to have.

Reply:
Oh sorry I had to do a bunch of merge conflict resolutions lol, it must've been caught in the cross-fire.